lib/thrift/generator/server/binary_framed.ex

```diff
@@ -94,6 +95,7 @@ defmodule Thrift.Generator.Server.BinaryFramed do
       Thrift.TApplicationException ->
         quote do
           unhandled ->
+            Logger.error("Unhandled exception: #{Exception.message(unhandled)}")
```

Comment:
Well, it's not really a TApplicationException, it's an exception you didn't handle properly.

"Exception was thrown but not declared in your thrift spec?"